iceoryx_posh/include/iceoryx_posh/internal/roudi/introspection/fixed_size_container.hpp

```diff
@@ -28,6 +28,8 @@
 // access to indices < 0 or >= capacity is undefined behaviour (most likely a
 // segmentation fault)
 
+using Capacity_t = uint64_t;
```

Comment:
I just noticed that the class is not in the iox namespace. Could you add the namespace or create an issue for this?

Also, to prevent namespace pollution, could you move the alias into the class since it's not used outside of the class. You could also use `decltype` on the capacity template parameter to infer the type.